### PR TITLE
Changed implementation on rounding up total pages

### PR DESCRIPTION
**Pull request by cedricabuso**

### CODE REVIEW

File: js/lfg.js [hunk 1]
```diff
@@ -68,9 +68,10 @@
                     }
 
                     if (!match_binded) {
+                        console.log(~~(data.total / data.limit));
```
Comment (nhvu1988) [inline js/lfg.js]: remove logging?

Comment (cedricabuso) [inline js/lfg.js]: Oh right, sorry :)

**Comment by cedricabuso** [comment]
@nhvu1988 Updated :)

**Comment by nhvu1988** [comment]
@ravenjohn checked, please merge it :)

**Pull request merged**